packages/admin/src/routes/entities/components/EditView/EditView.js

```diff
@@ -28,6 +28,11 @@ const EditView = props => {
     }
   }
 
+  const handleEntityDeleted = () => {
```

Comment:
what will happen if the detail is in a dirty state?

Reply:
good point.

In this case, the same notification is displayed as when you're trying to navigate away manually ("... are you sure you want to leave ..?"). If you deny that question, you stay on the detail form of the deleted entity.

What would be the desired behavior in your opinion?

Reply:
The prompt makes no sense since the current entity is already deleted, right? Hence validation will immediately throw errors when the users stays on the page. But i think it can be tricky to ignore the prompt in this case. Or better said: not worth the effort for a legacy action?